next/package.json

```diff
@@ -18,6 +18,7 @@
     "@azure/msal-node": "^1.12.0",
     "@cloudflare/json-schema-walker": "^0.1.1",
     "@js-joda/core": "5.2.0",
+    "@react-aria/button": "^3.6.1",
```

Comment:
no need for this if you're adding the whole `react-aria` library - you can import the button hooks directly from there

please remove this package and do so